index.html

```diff
@@ -59,7 +60,26 @@
     </header>
     <aside class="month-aside">
       <button class="add-event-button">Create Event</button>
-      <p>This is aside, here calendar will be inserted</p>
+      <div class="calendar-widget">
+       
+        <div class="calendar-widget-header">
+          <span id="currentDateCalendarWidget" class="calendar-widget-current-day"></span>
+          <div class="calendar-widget-controls-group">
+            <button id="prevMonth" class="switch-weeks-widget">&lt;</button>
+            <button id="nextMonth" class="switch-weeks-widget">&gt;</button>
+          </div>
+        </div>
+        <div class="calendar-widget-days">
```

Comment:
when reusing styles make sure that they fit your purpose. eg. Mo, Tu, ... now have cursor pointer and that suggests that they are clickable